source/administration/changelog.md

```diff
@@ -2,6 +2,215 @@
+
+### Improvements
+
+**Users**
+
+- After account creation, users are automatically directed to the team where they were invited instead of the Team Selection page.
+- Users can now be removed from teams via the **Main Menu** > **Manage Members** modal.
+
+
+**System Console**
+
+- Updated labeling of System Console settings in the UI for consistency and accuracy.
+- (Enterprise) Help text was added to the **Reload Configuration from Disk** and **Recycle Database Connections** buttons to explain their use in the switching databases with no down time.
+- Allow System Admins to create teams even if team creation is disabled via the System Console.
+
+**Notifications**
+
+- Address displayed in the email notification footer is now configurable in the System Console.
+- Direct Message desktop notifications now display with a "Direct Message" title.
+
+**Web UI**
+
+- Reply button and [...] menu now appear in a hovering UI element to increase the available margin width in the center channel.
+- Right-hand sidebar can now be expanded when viewing threads or search results.
+- Text emoticons now show up as the first entries in the autocomplete list
+- Added an online indicator to the header of Direct Message channels.
+- Added database type to the About Mattermost dialog.
+- Removed unnecessary resizing when opening and closing the right hand sidebar.
+- Removed jumping of the center channel when new messages are posted.
+- Updated the channel info dialog to be more user friendly.
+
+**Enterprise**
+
+- [New command line tools](http://docs.mattermost.com/administration/command-line-tools.html) added, such as inviting and removing users from channels, and restoring previously deleted channels.
+- Added a button to manually trigger LDAP synchronization.
+- Updating LDAP Synchronization Interval no longer requires a server restart to take effect.
+- Improved logging for LDAP synchronization.
+- Added validation to the LDAP settings in the System Console so an error is triggered if required fields are missing.
+
+### Bug Fixes
+
+- Privacy settings in the system console now refresh correctly when hiding email addresses or full names.
+- Fixed the cross contamination of new channels created on different teams in the same browser.
+- Updated the GitLab SSO error message for clarity if another Mattermost account is already associated with the GitLab account.
+- Team creation via GitLab SSO no longer throws an error if email domains are restricted.
+- Channel header no longer disappears after renaming a channel
+- Testing the email connection in the System Console no longer throws an error.
+- Multiline list items are now displayed correctly on new lines.
+- Error message is updated when switching from email to GitLab SSO authentication that is already used by another account.
+- Timestamps no longer require a page refresh when switching between 12h and 24h display formats.
+- `#in`, `#from`, or `#channel` now highlight in search results.
```

Comment:
This should be in hashtags **starting** with #in, #from, #channel (#in is not a valid hashtag, and #from seems to bring up no results)